Sources/Test/Directory.Build.props

```diff
@@ -1,110 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- Import parent Directory.Build.props if present -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   
-  <!-- This is needed for all .NET Framework projects that have a project reference to a multi-platform project: https://github.com/NuGet/Home/issues/4488 -->
-  <ItemGroup>
-    <PackageReference Include="Legacy2CPSWorkaround" Version="1.0.0" PrivateAssets="All" />
-  </ItemGroup>
-  
-  <!-- Setup global MSBuild variables -->
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <Optimize>false</Optimize>
```

Comment:
Is this specific to `Debug` version ?

Reply:
This is just for test code, so I think it's fine to leave unoptimized for all flavors

---
In reply to: [386609180](https://github.com/microsoft/Trill/pull/131#discussion_r386609180) [](ancestors = 386609180)